acceptance/terrafarm/farmer.go

```diff
@@ -271,6 +281,26 @@ func (f *Farmer) Assert(t *testing.T) {
 	}
 }
 
+// AssertStates verifies that the cluster state matches the provided map
+// of process names (e.g. cockroach) to expected states (e.g. RUNNING).
+func (f *Farmer) AssertStates(t *testing.T, procToExpState map[string]string) {
```

Comment:
You could also lose that map with my suggestion - simply check that everything is up everywhere.